src/isPalindrome.js

```diff
@@ -19,7 +19,8 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  return str.toLowerCase().replace(/[^a-z]/g, '')
```

Comment:
I think better to create a variable with result of this operations `str.toLowerCase().replace(/[^a-z]/g, '')` and then use it